# This expresses a marker/fiducial in a 3D space
int32 id                  # possible ids of the detected fiducial (empty if no id could be identified)
geometry_msgs/Pose avg_pose      # pose of the detected marker/fiducial